tests/integration/aws/general/package/tests.js

```diff
@@ -30,7 +29,8 @@ describe('AWS - General: Deployment with --noDeploy', function () {
     const deployedFiles = fs.readdirSync(path.join(process.cwd(), '.serverless'));
     expect(deployedFiles[0]).to.equal('cloudformation-template-create-stack.json');
     expect(deployedFiles[1]).to.equal('cloudformation-template-update-stack.json');
-    expect(deployedFiles[2]).to.match(/service-[0-9]{13}.zip/);
+    // Note: noticed the seconds section can vary a lot
+    expect(deployedFiles[2]).to.match(/test-[0-9]{1,}-[0-9]{9}.zip/);
```

Comment:
Unfortunately not since to make sure they are now generated on a timestamp instead of the service name.